SPECS/iscsi-initiator-utils/iscsi-initiator-utils.signatures.json

```diff
@@ -2,6 +2,6 @@
  "Signatures": {
   "04-iscsi": "7973dbc4f86506289fcf64255475598a9c6e153465c73633589961d96d2621af",
   "iscsi-tmpfiles.conf": "fd7766bfba489e4de4cc57713014c2a92c06164b49b135b97c8da679832e7146",
-  "open-iscsi-6.2.1.4+20210729.2a8f9d8.tar.gz": "a5a73f7c795d1dc214100b1eb70647d6c2ceccf8973e584f50ee7f92c40d2a8a"
+  "open-iscsi-test-2.1.9.tar.gz": "fd93e4e1a36236f7ec8b54859b386f3be940e899988a257ec87663b187c6ba63"
```

Comment:
i uploaded the wrong commit to open-iscsi-2.1.9.tar.gz.  need to figure out how to replace this or pick a better replacement name than *test*.

Reply:
using commit-based name unless i can replace the existing 2.1.9 withtout the commit.